<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEditSysinfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>431</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxPort">
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM8</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxBaud">
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSend"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSend">
      <property name="text">
       <string>循环发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelConnectionStatus">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>未连接</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>258</width>
     <height>376</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEditData1">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditData2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditData3">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditData4">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditData5">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>254</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButtonOpen">
      <property name="text">
       <string>打开端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonClose">
      <property name="text">
       <string>关闭端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonClearLog">
      <property name="text">
       <string>清除日志</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
